pkg/metrics/bundledeployment_metrics.go

```diff
@@ -0,0 +1,77 @@
+package metrics
+
+import (
+	fleet "github.com/rancher/fleet/pkg/apis/fleet.cattle.io/v1alpha1"
+	"github.com/rancher/fleet/pkg/summary"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	bundledeploymentSubsystem = "bundledeployment"
+	bundledeploymentLabels    = []string{"name", "namespace", "cluster_name", "cluster_display_name"}
+
+	bundleDeploymentState = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Subsystem: bundledeploymentSubsystem,
+			Name:      "state",
+			Help:      "Shows the state of this bundle deployment. Ready = 1, NotReady = 2, Pending = 3, OutOfSync = 4, Modified = 5, WaitApplied = 6, ErrApplied = 7.",
+		},
+		bundledeploymentLabels,
+	)
+
+	bundleDeploymentObserved = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: bundledeploymentSubsystem,
+			Name:      "total_observations",
```

Comment:
The suffix `_total` or `_count` is expected here. See https://prometheus.io/docs/instrumenting/writing_exporters/#naming. Same change in the above collector.

Reply:
Thank you for the link, updated!